rebasehelper/application.py

```diff
@@ -678,11 +676,29 @@ def get_rebased_patches(self):
             output_patch_string.append('Patches were not touched. All were applied properly')
         return output_patch_string
 
-    def print_summary(self):
-        output = output_tool.OutputTool(self.conf.outputtool)
-        report_file = os.path.join(self.results_dir, self.conf.outputtool + settings.REBASE_HELPER_OUTPUT_SUFFIX)
-        output.print_information(path=report_file)
-        logger.info('\nReport file from rebase-helper is available here: %s', report_file)
+    def print_summary(self, message=None):
+        """
+        Save rebase-helper result and print the summary using output_tools_runner
+        :param message: Error message from rebase-helper
+        :return:
+        """
+        log = None
+        # Store rebase helper result message
+        if message:
+            if len(message.args) > 1:
+                result = message.args[0] % message.args[1:]
+                if os.path.exists(message.args[2]):
```

Comment:
This is potential out-of-bounds access. How do you know that a third argument is a log file anyway?